### DISCUSSION TITLE
GIM Group Bank Search

### DISCUSSION THREAD

**Discussion by ForTimeBeing** [runelite/runelite#16319] [Ideas]
**Is your feature request related to a problem? Please describe.**
A clear and concise description of what the problem is. Ex. I'm always
frustrated when [...]

I have to constantly ask my group members if they have a certain item

**Describe the solution you'd like**

A way to search my group members banks remotely. (Eg. banked xp plugin remembers bank items even after leaving. I am sure there are external bank search plugins as well)

**Comment by barthogenes** [comment]
I use this plugin for my group:
https://github.com/christoabrown/group-ironmen-tracker
